shell/package.json

```diff
@@ -49,6 +49,7 @@
     "@nuxtjs/style-resources": "1.2.1",
     "@nuxtjs/webpack-profile": "0.1.0",
     "@popperjs/core": "2.4.4",
+    "@rancher/components": "0.1.0",
```

Comment:
Current version is `0.1.1`. As this seems a nuisance for each bump, should we think about not pin the versions or what would you recommend?